microsoft-edge/devtools-guide-chromium/remote-debugging/local-server.md

```diff
@@ -23,6 +23,8 @@ ms.date: 05/11/2021
    limitations under the License.  -->
 # Access local servers
 
+<!-- # Access local servers using the Remote devices panel/tab/tool -->
```

Comment:
If this commented-out line is really needed, maybe add a bit of a comment before to explain what it's for. Without context, it might be hard to remember why this comment is here in the future.

Reply:
removed cmt